types/src/edited_collection.rs

```diff
@@ -0,0 +1,47 @@
+use crate::CollectionSection;
+use crate::{Collection, Uuid};
+use std::option::Option;
+use {
+    crate::async_graphql::{self, dataloader::DataLoader, Context, FieldResult},
+    crate::Database,
+};
+
+/// Structure to represent an edited collection. Missing certain fields and chapters in it.
+/// Used for sending data to the front end
+#[derive(Debug, Clone, async_graphql::SimpleObject)]
+#[graphql(complex)]
+pub struct EditedCollection {
```

Comment:
Move these structs into the `collection.rs` file. Then put the ones used for data ingestion inside a `raw` submodule. Something like this:

```rs
struct EditedCollection { ... };
struct CollectionChapter { ... };

enum CollectionSection { ... };

mod raw {
    struct EditedCollection { ... };
    struct CollectionChapter { ... };
}
```

Match the `raw::` names like that so it's clear that they're semantically related. Then for usages we'll either see `EditedCollection` or `raw::EditedCollection`.